packages/flutter/test/cupertino/scaffold_test.dart

```diff
@@ -554,6 +554,6 @@ void main() {
     expect(richTextList.length, greaterThan(0));
     expect(richTextList.any((RichText text) => text.textScaleFactor != 1), isFalse);
 
-    expect(tester.widget<RichText>(find.descendant(of: find.text('content'), matching: find.byType(RichText))).textScaleFactor, 99);
+    expect(tester.widget<RichText>(find.descendant(of: find.text('content'), matching: find.byType(RichText))).textScaler, const TextScaler.linear(99.0));
```

Comment:
I appreciate all of the times in this PR that you added `.0` to a double.  Doubles without `.0` is a pet peeve of mine.